R/item_append_file.R

```diff
@@ -0,0 +1,30 @@
+#'@title Upload File to Item
+#'@description Adds a file to an item
+#'
+#'@param id A ScienceBase item ID to upload to.
+#'@param filename A file path to upload.
+#'@param sb_session A ScienceBase session object from authenticate_sb.
+#'
+#'@return ScienceBase Item.
+#'
+#'@import httr
```

Comment:
naw, skip it. Luke just said he uses 80% of httr anyways, so it doesn't matter. Just makes sense most of the time to avoid namespace conflicts